modules/acs-requirements.adoc

```diff
@@ -25,10 +29,14 @@ For example, Intel processors older than _Sandy Bridge_ and AMD processors older
 To install {product-title}, you must have:
 
 ifdef::openshift[]
-* {ocp} version {ocp-supported-version} or later
+* {ocp} version {ocp-supported-version} or later. For more details about supported self-managed and managed {ocp} versions, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
 endif::openshift[]
-* Cluster nodes with a supported operating system.
-For more information, see the link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+
+ifdef::other-platforms[]
+* A supported managed Kubernetes platform. For more details, see link:https://access.redhat.com/node/5822721[{product-title} Support Policy].
+endif::other-platforms[]
+
+* Cluster nodes with a supported operating system:
 ** *Operating system*: Amazon Linux, CentOS, Container-Optimized OS from Google, {op-system-first}, Debian, {op-system-base-full}, or Ubuntu.
```

Comment:
The problem with this operating systems list is that it shows up both for OCP secured cluster installations
1. https://60483--docspreview.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/prerequisites-cloud-ocp.html
2. https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/prerequisites-ocp.html

and non-ocp secured cluster installations
3. https://60483--docspreview.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/prerequisites-cloud-other.html
4. https://60483--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/prerequisites-other.html

The problem is that OCP itself is not supported on anything other than RHCOS and RHEL.